quickstart/src/main/res/menu/menu_video_activity.xml

```diff
@@ -3,12 +3,12 @@
     xmlns:android="http://schemas.android.com/apk/res/android">
 
     <item android:id="@+id/menu_turn_speaker_off"
-        android:title="@string/turn_speaker_off"
+        android:title="@string/turn_speaker_on"
```

Comment:
This is a little confusing. Shouldn't the string resource name match the id name?